core/handler/device/converter.go

```diff
@@ -0,0 +1,78 @@
+// Copyright © 2017 The Things Network
+// Use of this source code is governed by the MIT license that can be found in the LICENSE file.
+
+package device
+
+import (
+	pb "github.com/TheThingsNetwork/ttn/api/handler"
+	pb_lorawan "github.com/TheThingsNetwork/ttn/api/protocol/lorawan"
+)
+
+//Convert a device representation to a protoBuffer device struct
+func (dev *Device) ToPb() *pb.Device {
+	return &pb.Device{
+		AppId:       dev.AppID,
+		DevId:       dev.DevID,
+		Description: dev.Description,
+		Device: &pb.Device_LorawanDevice{LorawanDevice: &pb_lorawan.Device{
+			AppId:                 dev.AppID,
+			AppEui:                &dev.AppEUI,
+			DevId:                 dev.DevID,
+			DevEui:                &dev.DevEUI,
+			DevAddr:               &dev.DevAddr,
+			NwkSKey:               &dev.NwkSKey,
+			AppSKey:               &dev.AppSKey,
+			AppKey:                &dev.AppKey,
+			DisableFCntCheck:      dev.Options.DisableFCntCheck,
+			Uses32BitFCnt:         dev.Options.Uses32BitFCnt,
+			ActivationConstraints: dev.Options.ActivationConstraints,
+		}},
+		Latitude:  dev.Latitude,
+		Longitude: dev.Longitude,
+		Altitude:  dev.Altitude,
+		Builtin:   dev.Builtin,
+	}
+}
+
+//FromPb create a new Device from a protoBuffer Device
```

Comment:
- Missing space after `//`. Please check for this in other places as well
- create**s**. Also check this in other places